src/content/peerconnection/codec_constraints/js/test.js

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
Is this file needed?
It seems identical to https://github.com/webrtc/samples/blob/gh-pages/src/content/peerconnection/pc1/js/test.js.